docs/model_zoo.md

```diff
@@ -127,6 +128,42 @@ mxnet-model-server --models caffenet=https://s3.amazonaws.com/model-server/model
 curl -X POST http://127.0.0.1:8080/caffenet/predict -F "data=@kitten.jpeg"
 ```
 
+## <a name="duc-resnet101_onnx"></a>DUC-ResNet101 (from ONNX model zoo)
+* **Type**: Semantic Segmentation model trained on the Cityscapes dataset (model imported from ONNX)
+
+* **Reference**: [Wang et al.](https://arxiv.org/abs/1702.08502)
+
+* **Model Service**: [duc_service.py](https://s3.amazonaws.com/mxnet-model-server/onnx-duc/duc_service.py)
+
+* **Install dependencies**:
+```bash
+pip install opencv-python
+pip install pillow
+```
+
+* **Start Server**:
+```bash
+mxnet-model-server --models duc=https://s3.amazonaws.com/mxnet-model-server/onnx-duc/ResNet_DUC_HDC.model
+```
+
+* **Get the test image**:
+```bash
+curl -O https://s3.amazonaws.com/mxnet-model-server/onnx-duc/city1.jpg
+```
+
+* **Download inference script**:
+
+The script makes an inference call to the server using the test image, displays the colorized segmentation map and prints the confidence score.
+
```

Comment:
space here and no space above, make it consistent

Reply:
done